views/index.html

```diff
@@ -1,6 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
+    <meta charset="utf-8">
```

Comment:
Well, technically speaking, this is right, because there was a missing charset on this html page. Nice eye to see it!